RELEASE.md

```diff
@@ -0,0 +1,29 @@
+Release 1.2.1 of WebRTC C SDK
+
+Release tagged at: <Commit:  <to-be-filled>
+
+Whats’s new:
+- ICE server stats added
+- RTP outbound and inbound stats added (open source contribution)
+- Signaling stats available
+- Clang check added to Travis
+- MBedTLS support added. Choice between OpenSSL and mbedTLS available
+
+Improvements:
+- ICE candidate priority calculation bug fixed improving the candidate nomination process
+- Improved SDP conformance with RFC to work with CRLF and LF
+- Fixed interruption of sending process due to momentary network unreachable errors (open source contribution)
```

Comment:
We should add a line about fix for high CPU usage issues

Reply:
Will add once PR #671 is in.